chirho/explainable/handlers/preemptions.py

```diff
@@ -53,11 +58,13 @@ def __init__(
         *,
         prefix: str = "__witness_split_",
         bias: float = 0.0,
+        cases: Optional[Mapping[str, torch.Tensor]] = None,
```

Comment:
As with `SearchForExplanation`, it's hard for me (and presumably anyone less familiar with the code) to understand this complex new piece of the `Preemptions` interface, especially given the brief and cryptic documentation provided. Is there more than one use case for it, or even more than one correct set of values in the context of a given `SearchForExplanation` handler? If not, shouldn't it be our job to deal with that behind the scenes and avoid exposing it to users entirely?